web/impact/impact/tests/test_graphql.py

```diff
@@ -396,3 +397,29 @@ def test_non_staff_user_can_access_finalist_graphql_view(self):
                     }
                 }
             )
+
+    def test_get_user_confirmed_mentor_program_families(self):
```

Comment:
Can we also test the empty case? What if there are no confirmed mentor program families for this user?